app/helpers/view_helper.rb

```diff
@@ -22,4 +22,28 @@ def search_ui_url(relative_path)
   def search_ui_course_page_url(provider_code:, course_code:)
     search_ui_url("/course/#{provider_code}/#{course_code}")
   end
+
+  def course_has_unpublished_changes(course)
+    course.attributes[:content_status] == "published_with_unpublished_changes"
+  end
+
+  def course_content_status(course)
+    status = course.attributes[:content_status]
```

Comment:
If you only need the status, you could just pass in the status? Both works though :)

Reply:
I like passing in the course so that the helper does the digging, but yeah either is ok :)